src/assets/js/admin/settings/common/doUploadListener.js

```diff
@@ -29,7 +29,11 @@ export function doUploadListener () {
       button: {
         text: $button.data('uploader-button-text')
       },
-      multiple: false
+      multiple: false,
+
+      library: {
+        type: [ 'font/ttf', 'application/x-font-ttf', 'application/octet-stream' ]
+      }
```

Comment:
This component isn't limited to the Font Manager. Causing issues with image uploads.